packages/ui/package.json

```diff
@@ -167,4 +167,4 @@
       "prettier ./src --write"
     ]
   }
-}
+}
```

Comment:
pls avoid unnecessary changes. let's revert this

Reply:
Hey @abhinavrastogi-harness 
Thanks for pointing out this✌️
I had the wrong default formatter in my IDE, fixed it and reverted the change as well👌